qtpy/uic.py

```diff
@@ -226,3 +228,46 @@ def loadUi(uifile, baseinstance=None, workingDirectory=None):
         widget = loader.load(uifile)
         QMetaObject.connectSlotsByName(widget)
         return widget
+
+
+    # Credit:
+    # http://stackoverflow.com/questions/4442286/python-code-genration-with-pyside-uic/14195313#14195313
```

Comment:
Please put this comment inside the function docstring and use the Stackoverflow post shorten version link: https://stackoverflow.com/a/14195313/15954282

Reply:
done